src/components/CategoryIndicator/CategoryIndicator.scss

```diff
@@ -76,13 +70,3 @@
   font-size: 12px;
   font-weight: 700;
 }
-
-.viewIcon {
-  // TODO (davidg): icon and combine all icons
```

Comment:
In SC-16 you centralised these, but I've removed them all completely, so you can get rid of that mixin from SC-16 if you want.